src/utils/parameters.py

```diff
@@ -1,5 +1,8 @@
 import os
 import argparse
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
are these necessary? the one in deadbot.py should load your env

Reply:
Isn't necessary on my desktop, but it might've been on my laptop. I can't remember, but your probably right it should only need the one in deadbot.py. I'll remove it from here for now and if issues arise I'll fix it elsewhere.